### DISCUSSION TITLE
A request for an online environment or documents

### DISCUSSION THREAD

**Discussion by qazmkop** [lakesoul-io/LakeSoul#15] [Q&A] [answered]
I have an urgent project to use the CDC  of Lakesoul, but the setup environment is too complicated and will take a lot of time. Does Lakesoul have an online environment or documents for users?

**Comment by dmetasoul01** [comment, marked as answer]
Thanks for the feedback. We already have a [Quick Start](https://github.com/meta-soul/LakeSoul/wiki/02.-QuickStart) doc on how to setup a basic environment for testing. We'll provide an end-to-end CDC demo. See #16
